github.go

```diff
@@ -76,6 +79,11 @@ func NewGitHubClient(source Source) (*GitHubClient, error) {
 		if err != nil {
 			return nil, err
 		}
+
+	}
+
+	if source.GitHubV4APIURL != "" {
```

Comment:
If `github_api_url` is set but `github_v4_api_url` is not, I think it might make sense to default `source.GithubV4APIURL` to `source.GithubAPIURL + "/graphql"`. That way, people who are using enterprise GitHub won't need to change their `source` config at all, unless they have a different `/graphql` endpoint on their enterprise server.